SharedSource/Development/UnitTesting/PEVerifyPathSources/CompoundPEVerifyPathSource.cs

```diff
@@ -33,9 +33,9 @@ public CompoundPEVerifyPathSource (params IPEVerifyPathSource[] sources)
       _sources = sources;
     }
 
-    public string? GetPEVerifyPath (PEVerifyVersion version)
+    public string GetPEVerifyPath (PEVerifyVersion version)
```

Comment:
This remains nullable. We're using FirstOrDefault instead of First so the nullability is part of the game here.